main.js

```diff
@@ -0,0 +1,162 @@
+var _startX = 0;            // mouse starting positions
+var _startY = 0;
+var _offsetX = 0;           // current element offset
+var _offsetY = 0;
+var _dragElement;           // needs to be passed from OnMouseDown to OnMouseMove
+var _oldZIndex = 0;         // we temporarily increase the z-index during drag
+var iterator = 10;
+
+var premadePhrases = {
+    sample:"the,quick,brown,fox,jumps,over,the,lazy,dog",
+    lolcat:"lol,cat,lolcat,iz,r,ur,good,?,i,want,cheezburger,nao",
+    geek:"battle,awesome,are,an,action,could,complete,cheat,bit,force,information,is",
+    bacon:"the,day,begin,s,with,this,morning,marvel,bring,ing,the,aroma,of,satisfaction,bacon",
+    beard:"my,beard,is,a,lifestyle,facial,hair,touch,it,thanks",
+    explicit:"fuck,fuck,fuck"
+};
+
+
+function initDragDrop() {
+    document.addEventListener('mousedown', onMouseDown);
+}
+
+initDragDrop();
```

Comment:
I'm not sure the function is necessary, you're not reusing it so line 20 can just be on it's own.